### ISSUE TITLE
Building Error in the latest Wasmtime

### ISSUE THREAD

**Issue by hungryzzz** [bytecodealliance/wasmtime#6870]
Hi, I try to build the latest Wasmtime on my local, and I meet the building error as follow,

```error: Package `wasmtime v13.0.0 (/home/ringzzz/wasm_runtime/wasmtime/crates/wasmtime)` does not have feature `wasmtime-cranelift`. It has an optional dependency with that name, but that dependency uses the "dep:" syntax in the features table, so it does not have an implicit feature with that name.```

The building commands are as follow:
```bash
git clone git@github.com:bytecodealliance/wasmtime.git
cd wasmtime
git submodule update --init
cargo build --release # raise the building error
```

### Hardware & OS
- Ubuntu 20.04
- CPU: Intel(R) Core(TM) i5-9500T CPU @ 2.20GHz

**Comment by bjorn3** [comment]
Which rustc version do you use?

**Comment by hungryzzz** [comment]
@bjorn3  rustc 1.67.1 (d5a82bbd2 2023-02-07)

**Comment by alexcrichton** [comment]
I believe you're running into https://github.com/rust-lang/cargo/pull/12130 which was first released in Rust 1.71.0. Locally I see the same error using Rust 1.70 and prior. Are you able to update your rustc version?
